project/CopyrightHeader.scala

```diff
@@ -33,7 +33,7 @@ object CopyrightHeader extends AutoPlugin {
 
   // We hard-code this so PR's created in year X will not suddenly fail in X+1.
   // Of course we should remember to update it early in the year.
-  val CurrentYear = "2020"
+  val CurrentYear = "2021"
```

Comment:
use Date?

Reply:
As noted in the comment above this line, we prefer to hard-code the year, because otherwise all PR's that are open on December 31st and master suddenly start failing on January 1st